src/pyfme/simulator.py

```diff
@@ -5,276 +5,331 @@
-                         'rho': self.environment.rho,
-                         'a': self.environment.a,
-                         'TAS': self.aircraft.TAS,  # aircraft
-                         'Mach': self.aircraft.Mach,
-                         'q_inf': self.aircraft.q_inf,
-                         'alpha': self.aircraft.alpha,
-                         'beta': self.aircraft.beta,
-                         'x_earth': self.system.x_earth,  # system
-                         'y_earth': self.system.y_earth,
-                         'z_earth': self.system.z_earth,
-                         'psi': self.system.psi,
-                         'theta': self.system.theta,
-                         'phi': self.system.phi,
-                         'u': self.system.u,
-                         'v': self.system.v,
-                         'w': self.system.w,
-                         'v_north': self.system.v_north,
-                         'v_east': self.system.v_east,
-                         'v_down': self.system.v_down,
-                         'p': self.system.p,
-                         'q': self.system.q,
-                         'r': self.system.r,
-                         'height': self.system.height,
-                         'F_xb': self.aircraft.total_forces[0],
-                         'F_yb': self.aircraft.total_forces[1],
-                         'F_zb': self.aircraft.total_forces[2],
-                         'M_xb': self.aircraft.total_moments[0],
-                         'M_yb': self.aircraft.total_moments[1],
-                         'M_zb': self.aircraft.total_moments[2]
-                         }
-        for par_name, par_values in self.par_dict.items():
-            par_values[self._time_step] = self.PAR_KEYS[par_name]
-
-
-class RealTimeSimulation(Simulation):
-
-    def __init__(self, aircraft, system, environment):
-        raise NotImplementedError()
-        super(RealTimeSimulation, self).__init__(aircraft, system, environment)
-        # TODO:...
-
-    def _get_current_controls(self, ii):
-        # Joystick reading
-        raise NotImplementedError
-
-    def set_par_dict(self, par_list):
-
-        for par_name in par_list:
-            if par_name in self.PAR_KEYS:
-                self.par_dict[par_name] = []
+
+        system = self.system
+        env = self.environment
+        ac = self.aircraft
+
+        # Initialize state
+        system.set_initial_state(
+            geodetic_coord=geodetic_initial_pos,
+            euler_angles=np.array([0, 0, psi])
+                                 )
+        # Update environment for the current state
+        env.update(system)
+
+        # Initialize alpha and beta
+        # TODO: improve initialization method
+        alpha0 = 0.05
+        beta0 = 0.001 * np.sign(turn_rate)
+
+        # For the current alpha, beta, TAS and env, set the aerodynamics of
+        # the aircraft (q_inf, CAS, EAS...)
+        ac._calculate_aerodynamics_2(TAS, alpha0, beta0, env)
```

Comment:
Oh, it's used here. It didn't show when `Ctrl+F` because the diff was collapsed.